README.md

```diff
@@ -58,21 +58,21 @@ The basic Meeshkan flow is **collect, build and mock.**
 
 ### Tutorial
 
-The quickest way to get an overview of Meeshkan is to complete our [interactive tutorial](https://github.com/meeshkan/meeshkan-tutorial). It walks you through the collect, build, and mock flow - while also covering the concepts necessary for development.
+The quickest way to get an overview of Meeshkan is to complete our interactive tutorial. It walks you through the collect, build, and mock flow - while also covering the concepts necessary for development.
 
-First, install `meeshkan-tutorial` via [pip](https://pip.pypa.io/en/stable/installing/):
+First, install `meeshkan` via [pip](https://pip.pypa.io/en/stable/installing/):
 
 ```bash
-$ pip install meeshkan-tutorial
-$ meeshkan-tutorial
+$ pip install meeshkan
+$ meeshkan tutorial
 ```
```

Comment:
I'd remove all of this since it comes right after the installation section (the only reason we had it originally was because you needed to install something different).